docs/only-allow-admins-to-create-new-streams-feature.md

```diff
@@ -0,0 +1,12 @@
+# Only allow admins to create new streams
+
+1. Login with an Admin account.
```

Comment:
This step is unnecessary. You should've mentioned that this option is only available to administrators in your previous one-sentence summary.